packages/app/src/pages/PostCall.tsx

```diff
@@ -4,14 +4,16 @@ import { useNavigate } from 'react-router-dom';
 import { CustomButton, CustomContainer, LoadingSpinner } from '../components';
 import { createPatientName, getRelativeTime } from '../helpers';
 import { useAuth0 } from '@auth0/auth0-react';
-import { useVideoParticipant } from '../store';
+import { useParticipant, useVideoParticipant } from '../store';
 
 // TODO: Merge 2 post call pages in one
-export const ProviderPostCall = (): JSX.Element => {
+export const PostCall = (): JSX.Element => {
   const { isAuthenticated } = useAuth0();
   const navigate = useNavigate();
   const { t } = useTranslation();
   const { callStart } = useVideoParticipant();
+  const { providerName } = useParticipant();
+  const isProvider = window.location.pathname === '/provider-post-call';
```

Comment:
not sure if this is a good way to detect if `isProvider`...
and better if we could have 1 path for both of patient and provider? like /post-call

Reply:
one way is:
add state to the navigation with react-router-dom
some example:

in video call page/component do:
```
    navigate('/post-call', {
      isProvider: true,
    });
```
and in post-call:
```
`const location = useLocation();`
const isProvider = location.state.isProvider;
```

Reply:
Doing one path for both Patient and Provider will result in authentication issue. We wrap all pages that are Provider specific (which include top app bar and dashboard button) with a private route which requires authentication and redirects to a login page when user logs out. In this case, if we have same path for both, either our patients will face authentication requirement, or provider will not be able to logout from the post call page.

Reply:
I will try to improve the way we detect `isProvider`, adding variable to navigation seems reasonable